### ISSUE TITLE
Plugin not working on Meta Quest 3

### ISSUE THREAD

**Issue by DomCoccorese** [labstreaminglayer/plugin-UE4#11]
Hello
I'm using this plugin inside an Unreal Engine project where I use an outlet component to send a simple message. Although it works in preview, the resulting apk installed in my Oculus Meta Quest 3 does not send any message. I want to specify that the Headset and the computer (where the inlet program is installed) are connected to the same network. Am I missing something? Does anyone faced the same issue with Meta Quest?

Thanks in advice

**Comment by tstenner** [comment]
Does the app have the `CHANGE_WIFI_MULTICAST_STATE` permission?

**Comment by DomCoccorese** [comment]
@tstenner thanks for the response. I've just tried your solution enabling CHANGE_WIFI_MULTICAST_STATE. Unfortunately it does not change anything

**Comment by brifsttar** [comment]
Did you make changes to the plugin so that it can build for Android? At the moment, only Windows is supported.

**Comment by DomCoccorese** [comment]
No I didn't make any change, I've just imported the plugin in Unreal and then built the project for Android. So this Plugin does not support Android? Is there a way to make it work? Thanks

**Comment by brifsttar** [comment]
This plugin only supports Windows, but it could probably work with Android if you have a `liblsl` built for Android (which I don't think are part of the release pipeline? But not sure).

If you have that, then you can probably edit the plugin to add Android support ([[0]](https://forums.unrealengine.com/t/including-shared-object-library-in-android/57840/2), [[1]](https://nerivec.github.io/old-ue4-wiki/pages/how-to-add-a-shared-library-so-in-android-project.html)).

In the meantime, I find it really weird that Unreal lets you package an Android project that uses this plugin without failing.

**Comment by DomCoccorese** [comment]
Yes I ensure that the plugin is enabled in Plugins tab and the package process completes successfully. I'm using Unreal 5.3.2. 
I'm already looking for liblsl (found here https://github.com/labstreaminglayer/liblsl-Android/tree/master), but for now I don't manage to build it (sorry but I'm not an Android Studio expert). In the links you provided it is reported that I need an *.so file to include, but I don't find any .so file inside liblsl repository
